src/main/java/io/github/nucleuspowered/nucleus/modules/mute/commands/MuteCommand.java

```diff
@@ -46,10 +47,12 @@
 @RegisterCommand({"mute", "unmute"})
 public class MuteCommand extends CommandBase<CommandSource> {
 
+    @Inject private MuteConfigAdapter mca;
     @Inject private UserDataManager userConfigLoader;
     @Inject private MuteHandler handler;
 
     private final String notifyPermission = PermissionRegistry.PERMISSIONS_PREFIX + "mute.notify";
+    private final String bypassMaximumLength = PermissionRegistry.PERMISSIONS_PREFIX + "mute.bypass";
```

Comment:
I need to fix the notify permission myself - but otherwise, this is the same as above.